src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs

```diff
@@ -104,6 +104,26 @@ public void OnRequestDownload(PackageVersion version, bool downloadToCustomPath)
                 RequestDownload(this.Model, version, downloadPath);
         }
 
+        /// <summary>
+        /// Comparator 
+        /// </summary>
+        /// <param name="other">The PackageManagerSearchElementViewModel to compare</param>
+        /// <returns></returns>
+        public bool Equals(PackageManagerSearchElementViewModel other)
+        {
+            if (other == null) return false;
+            return this.Model.Id == other.Model.Id && this.Model.Name == other.Model.Name;
+        }
+
+        /// <summary>
+        /// Overridden Getter for HashCode 
+        /// </summary>
+        /// <returns>HashCode of package</returns>
+        public override int GetHashCode()
+        {
+            return Model.Id.GetHashCode() + Model.Name.GetHashCode();
```

Comment:
I don't think this is a good way to implement a hashcode - where did you find this?

I would use `^` xor

https://stackoverflow.com/questions/9827911/how-to-implement-override-of-gethashcode-with-logic-of-overriden-equals

Reply:
@mjkkirschner Thanks, I realized we are discussing an overkill and I prefer to use just the `Model.Id.GetHashCode()` which I think is enough for this case